src/Desktop/UIAutomation/CustomObjects/Registrar.cs

```diff
@@ -58,6 +58,21 @@ public void RegisterCustomProperty(CustomProperty prop)
         public Dictionary<int, CustomProperty> GetCustomPropertyRegistrations() { return new Dictionary<int, CustomProperty>(_idToCustomPropertyMap);  }
 #pragma warning restore CA1024 // Use properties where appropriate: this is a copy of the object, not the object itself
 
+        ///  <summary>Notifies ConverterRegistrationAction about all of the custom property registrations in the passed-in dictionary. This is used, for instance, when switching between file and live mode in Accessibility Insights for Windows.</summary>
```

Comment:
General comments on summary blocks (some apply to previous changes that I didn't notice)
- VS typically uses 3 slashes, then a space, then the < sign. Many of these lines have extra spaces after the slashes
- Really long lines aren't helpful. Let's cap line length at less than 100 chars
- When VS adds these comments, the opening and closing tags are usually on separate lines. It seems OK to use the single-line form if the whole line is within the aforementioned 100 chars limit. If we need multiple lines, let's put the opening and closing tags on their own lines.